functions/loadSingleCommand.js

```diff
@@ -38,6 +48,15 @@ module.exports = (client, command, reload = false, loadPath = null) => new Promi
       pathParts = pathParts.slice(pathParts.indexOf("commands") + 1);
       category = client.funcs.toTitleCase(cmd.help.category ? cmd.help.category : (pathParts[0] && pathParts[0].length > 0 && pathParts[0].indexOf(".") === -1 ? pathParts[0] : "General"));
       subCategory = client.funcs.toTitleCase(cmd.help.subCategory ? cmd.help.subCategory : (pathParts[1] && pathParts[1].length > 0 && pathParts[1].indexOf(".") === -1 ? pathParts[1] : "General"));
+
+      codeLang = "JS";
+      compiledLangs.forEach((lang) => {
+        // Remove the ".js" extension, if there is one, since it's optional.
+        const compiledPath = `${loadPath.replace(/\.js$/, "")}.${lang.toLowerCase()}`;
+        // If there's an equivalent file that ends with the lang, it's a code
+        // file that was compiled into JS.
+        if (fs.existsSync(compiledPath)) codeLang = lang.toUpperCase();
```

Comment:
fs.existsSync is not deprecated. https://nodejs.org/dist/latest-v7.x/docs/api/fs.html#fs_fs_existssync_path
"Note that fs.exists() is deprecated, but fs.existsSync() is not."